### PR TITLE
High Sierra: update expected CLT/libcurl pkgconfig file.

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-----

Various changes per DP/Xcode/CLT updates released a couple days back.

### CODE REVIEW

File: Library/Homebrew/os/mac/xcode.rb [hunk 1]
```diff
@@ -216,7 +216,7 @@ def latest_version
         # on the older supported platform for that Xcode release, i.e there's no
         # CLT package for 10.11 that contains the Clang version from Xcode 8.
         case MacOS.version
-        when "10.13" then "900.0.22.8"
+        when "10.13" then "900.0.26"
```
Comment (mistydemeo) [inline Library/Homebrew/os/mac/xcode.rb]: Ooh, new clang build? Too bad it doesn't have the chance I'm really holding out for...

**Comment by MikeMcQuaid** [comment]
Holding off on this for a bit to give folks time to upgrade.

File: Library/Homebrew/os/mac/xcode.rb (hunk 1 above)
Comment (DomT4) [inline Library/Homebrew/os/mac/xcode.rb]: Aye! Didn't see any changes to the behaviour we were discussing the other day, but RL has prevented me actually being able to look particularly hard 😓.

**Comment by MikeMcQuaid** [comment]
Thanks again @DomT4!

**Comment by DomT4** [comment]
Thanks Mike. My VMWare trial license only has 6 days to run, so after that I guess I'll have to make a decision on whether to upgrade my main machine or let someone else take over these PRs.

**Pull request merged**